jog_arm/src/jog_arm/jog_arm_server.cpp

```diff
@@ -1001,6 +1016,11 @@ Eigen::MatrixXd JogCalcs::pseudoInverse(const Eigen::MatrixXd& J) const
   return J.transpose() * (J * J.transpose()).inverse();
 }
 
+Eigen::MatrixXd JogCalcs::pseudoInverse(const Eigen::MatrixXd& U_matrix, const Eigen::MatrixXd& V_matrix, const Eigen::MatrixXd& S_diagonals) const
```

Comment:
Might want to add a comment about the difference between these two pseudoInverse methods.